guide/locations/_azure-ARM.md

```diff
@@ -95,6 +95,17 @@ The loginUser can be anything, as long as it's specified.
 The `overrideAuthenticateSudo: true` key tells Apache Brooklyn that default on Azure images do not have passwordless sudo 
 configured by default.
 
+    imageId: westeurope/OpenLogic/CentOS/7.3
+
+Listing images in Azure CLI [https://docs.microsoft.com/en-us/cli/azure/vm/image#list-skus](https://docs.microsoft.com/en-us/cli/azure/vm/image#list-skus).
```

Comment:
Why should users use this over [cloud:list-images](http://docs.cloudsoftdev.io/operations/tools/index.html#cloud-compute-tools)